scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
This is code only run by generators in `scripts/`, it should be at the same place as generators. Maybe it would make sense to separate generators? it seems a very consistent and well-defined pattern, unlike some of the random stuff in `scripts`. So `scripts/generators/helpers.lua` ? Or just `generators/helpers.lua` ?